packages/blocks/src/components/blockhub.ts

```diff
@@ -586,6 +602,11 @@ export class BlockHub extends NonShadowLitElement {
     this.requestUpdate();
   };
 
+  private _onBlockHubEntryMouseOver = () => {
+    this._isCardListVisible = false;
```

Comment:
`_isCardListVisible` should be a `@state`, this avoids manual `requestUpdate`

Reply:
Well I took a look, in this component we have too many non `@state` variables, so let's keep this as is but here may need further refactoring.

Reply:
There seems to be more variables that can be changed to `@state` variables to reduce the usage of `requestUpdate`👀